### ISSUE TITLE
biallelic_PAR     

### ISSUE THREAD

**Issue by angharadroberts** [obophenotype/human-phenotype-ontology#7499]
**Preferred term label:**
biallelic_PAR     

**Synonyms**
PAR recessive inheritance (requested)

**Definition (free text, please give PubMed ID)**
The allelic requirement for PAR recessive inheritance

**Parent term (use hpo.jax.org/app)**
allelic requirement (requested)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added as allelic requirement synonym to the new term added as HP:0034341
